builds/securing-builds-by-strategy.adoc

```diff
@@ -0,0 +1,55 @@
+[id='securing-builds-by-strategy']
+= Securing builds by strategy
+include::modules/common-attributes.adoc[]
+:context: securing-builds-by-strategy
+toc::[]
+
+Builds in {product-title} are run in privileged containers. Depending on the
+build strategy used, this allows a user who can run builds to escalate their
+permissions on the cluster and host nodes. As a security measure, limit who can
+run builds and the strategy that is used for those builds. Custom builds are
+inherently less safe than Source builds, given that they can execute any code
+within a privileged container, and as such are disabled by default. Docker build
```

Comment:
s/, and as such/and

Reply:
Do we need more clarification about how the Docker builds work since Docker isn't available by default anymore?

Reply:
not really(at least not here).  that's why i wrote it as "dockerfile processing logic".  deliberately vague about implementation specifics.  We have the capability to turn your dockerfile into an image.  In 3.x we used the docker daemon, in 4.x we use buildah.  in 5.x....????

Reply:
That's fair. I'm thinking more a note that points out that it's called the Docker build because it uses the Dockerfile, but that in {product-title} 4, Docker is not required. Buildah does it.

Reply:
that makes sense to me.